src/compat.h

```diff
@@ -1,8 +1,12 @@
 #ifndef TIMESCALEDB_COMPAT_H
 #define TIMESCALEDB_COMPAT_H
 
-#define PG96 ((PG_VERSION_NUM >= 90600) && (PG_VERSION_NUM < 100000))
-#define PG10 ((PG_VERSION_NUM >= 100000) && (PG_VERSION_NUM < 110000))
+#define check_version_96(version) ((version >= 90603) && (version < 100000))
```

Comment:
Maybe a more clear name here is `is_supported_pg_version()`? It is not clear that `check_version` is a boolean statement as opposed to, e.g., throwing an exception or something else.